### PR TITLE
add java client usage in README

### PR DESCRIPTION
**Pull request by joejoe321321**

### CODE REVIEW

**Comment by yahoocla** [comment]
CLA is valid!

File: README.md [hunk 1]
```diff
@@ -14,10 +14,11 @@ The goal of Parsec is to:
 * Minimize time and effort spent on common repetitive tasks
 * Provide helper [libraries and utilities](https://github.com/yahoo/parsec-libraries) for common tasks
 * Reduce the learning curve and maintenance cost
+* Provide client for your web service
```
Comment (shichung) [inline README.md:17]: could it be modified to be more specific or clear?
e.g. Provide autogenerated Java client for your web service?

**Comment by shichung** [comment]
LGTM

**Comment by shichung** [comment]
does src/main/rdl/sample.rdl have to be synced back with the internal one?

File: README.md (hunk 1 above)
Comment (joejoe321321) [reply README.md:17]: Yes, but not necessary. Details are below.

**Comment by joejoe321321** [comment]
Yes, it will be more clear.

**Comment by shichung** [comment]
ok

**Pull request merged**